packages/server/.env.example

```diff
@@ -46,3 +46,9 @@ AUDIT_REPORT_BUCKET=arpa-audit-reports
 
 # SQS queue for grants-ingest events
 GRANTS_INGEST_EVENTS_QUEUE_URL="http://localhost:4566/000000000000/grants-ingest-events"
+
+# Datadog tracing configuration
+DD_ENV="sandbox"
```

Comment:
I think it would be ideal for people to use some kind of unique modifier rather than tagging everything as `env:sandbox`. For example, I've tended to use `env:sandbox-tsh` for local development, as it helps ensure that the data I'm looking at came from my own debugging activities and not from someone else's.

Reply:
I don't think this sample configuration precludes that workflow if there is ever any confusion, does it?  I also think it would just be _more_ confusing if I put my initials in here and someone copy-pasted the example configuration without editing.

Reply:
For sure – This comment was really just meant to contextualize my [other comment](https://github.com/usdigitalresponse/usdr-gost/pull/1841#discussion_r1307708551) related to the documentation – figured it might be worth adding a note around this in your markdown docs.